dashboard/client/src/pages/overview/OverviewPage.tsx

```diff
@@ -51,8 +51,8 @@ const useStyles = makeStyles((theme) =>
 export const OverviewPage = () => {
   const classes = useStyles();
 
-  const { cluster_status } = useRayStatus();
-
+  // const { cluster_status } = useRayStatus();
+  const cluster_status = undefined;
```

Comment:
revert this

Reply:
Ohh, this is not ready for review... Just try to reproduce the errors